src/com/google/enterprise/gsafeed/Metadata.java

```diff
@@ -43,14 +43,16 @@
   /**
    * Gets the value of the overwriteAcls property.
    *
-   * @return possible object is {@link String}
+   * @return possible object is {@link boolean}
    */
-  public String getOverwriteAcls() {
+  public boolean getOverwriteAcls() {
+    // The default as defined in the dtd is true. Return that
+    // here to reflect the underlying meaning, but don't set the
```

Comment:
This comment seems odd in the getter. What does it mean to not the set string here? Also, you're not allowing setOverwriteAcls(null) to reset the default, like you do for the enums. Maybe at least allow that for setOverwriteAcls(String), and maybe think about using Boolean instead of boolean, and live with the autoboxing?